subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/java/JavaLibraryPlugin.java

```diff
@@ -46,16 +48,38 @@ public void execute(CopySpec copy) {
             }
         });
 
+        final File dependenciesFile = new File(project.getBuildDir(), "dependency-info.json");
+
+        Task createDependencyInfoFileTask = TaskMaker.task(project, "createDependencyInfoFile", CreateDependencyInfoFileTask.class, new Action<CreateDependencyInfoFileTask>() {
```

Comment:
Nitpick: I suggest shorter variable names like "dependencyTask" or "dependencyInfoTask". The code is easier to read (less characters).